README.md

```diff
@@ -1 +0,0 @@
-# windows-terminal
```

Comment:
why are we cleaning up this line?

Reply:
Because we're going to do the docs in a second stage and we're not using headers to refer to our repos